.github/workflows/rspec.yml

```diff
@@ -18,10 +19,12 @@ jobs:
           - "2.6"
           - "2.7"
           - "3.0"
-          - "head"
           - "jruby"
+        include:
```

Comment:
I think it needs `experimental: [false]` above this include for the non-experimental rubies, I am not yet deep into the Github actions syntax. I have a working example here: https://github.com/WeTransfer/zip_tricks/blob/main/.github/workflows/ci.yml

Reply:
I see this one appears to have RSpec failed but the job succeeds. 
https://github.com/WeTransfer/zip_tricks/runs/1667923935?check_suite_focus=true

This is not quite the same as TravisCI allowed failures since the job shows failed there but the overall status is success. 

I'm not a fan of how GitHub set this up because it requires digging in to see the failure rather than seeing one red X.

Reply:
See https://github.com/actions/toolkit/issues/399#issuecomment-738700569

Reply:
agreed, it is indeed worse.